lib/driver.js

```diff
@@ -1588,9 +1588,21 @@ class XCUITestDriver extends BaseDriver {
     return true;
   }
 
-  async checkAutInstallationState() {
-    // @ts-expect-error - do not assign arbitrary properties to `this.opts`
-    const {enforceAppInstall, fullReset, noReset, bundleId, device, app} = this.opts;
+  /**
+   * @typedef {Object} CheckAutInstallationState
+   * @property {boolean} install
+   * @property {boolean} skipUninstall
+   * /
+
+  /**
+   * Check if the given app can be installed, or should uninstall before installing it.
+   *
+   * @param {XCUITestDriverOpts} [opts]
```

Comment:
Lets just only enumerate the options we consider in this method. Explicit is better than implciit